### PR TITLE
[WNMGDS-2186] Creating `info` color category and refining `primary` and `secondary`

### PR DESCRIPTION
**Pull request by pwolfert**
## Summary

https://jira.cms.gov/browse/WNMGDS-2186

Builds on https://github.com/CMSgov/design-system/pull/2381 and https://github.com/CMSgov/design-system/pull/2381 to better organize our theme colors and make room for color differences between themes by not making assumptions about how `secondary` is used.

- Added missing `primary` shades for each theme. Each theme was missing different shades. It now has a complete range from `*-lightest` to `*-darkest` like the other color variables.
- Created an `info` status color variable with `*-lightest` to `*-darkest`. This will be used for the info alerts and badges, which previously had to draw their colors from `secondary`. Using `secondary` for alerts was forcing us to make `secondary` a shade of blue for each theme, which was limiting.
- Assigned the green colors to `secondary` for core and healthcare, which actually use greens as their secondary color for buttons. Healthcare uses that secondary green for `main` buttons, and core uses the secondary green for `main` buttons.
- Removed the `green` color variable, which was now a duplicate of the semantic `success` color. This finally finished the work started in https://github.com/CMSgov/design-system/pull/2381 of removing the non-semantic color categories.
- Simplified the alternate button styles for healthcare since they don't use them. They are now duplicates of the main styles except for the solid buttons, since blue solid buttons are still in use in some applications
- Changed on-dark alternate color of buttons for the core theme to be green, which is consistent with the solid version
- Changed the doc-site nav panel color from secondary to primary for each theme

## How to test

[Demo link](https://cmsgov.github.io/design-system/branch/pwolfert/colors-info-primary-secondary)

### CODE REVIEW

**Comment by pwolfert** [comment]
Turns out the solid alt button on dark backgrounds is a different color now in core, so we need to address that before I can update the visual regression refs.

File: packages/design-system-tokens/src/themes/healthcare-component.ts [hunk 1]
```diff
@@ -239,10 +239,10 @@ export const components: AnyTokenValues = {
     '__border-color--disabled':                   'transparent',
     '__border-color--hover':                      'transparent',
     '__border-color':                             'transparent',
-    '__color--active':                            t.color['secondary'],
+    '__color--active':                            t.color['gray-lighter'],
     '__color--disabled':                          t.color['gray-dark'],
-    '__color--hover':                             t.color['secondary-light'],
-    '__color':                                    t.color['secondary-lighter'],
+    '__color--hover':                             t.color['gray-lightest'],
+    '__color':                                    t.color['white'],
```
Comment (pwolfert) [inline packages/design-system-tokens/src/themes/healthcare-component.ts:245]: Like I mentioned in the description, I simplified the unused healthcare button styles by copying the `main` to `alternate` styles. Below is a screenshot of the changes:

<details>
  <summary>Before/After screenshot</summary>
  
  healthcare-buttons-before-after
</details>

File: packages/design-system-tokens/src/themes/healthcare.ts [hunk 2]
```diff
@@ -75,13 +70,21 @@ export const themeColors: ColorTokens = {
...
-  'secondary-darker':           color['darksky-700'],
-  'secondary-darkest':          color['darksky-800'],
-  'secondary-light':            color['darksky-300'],
-  'secondary-lighter':          color['darksky-100'],
-  'secondary-lightest':         color['darksky-50'],
+  'secondary':                  color['spring-500'],
+  'secondary-dark':             color['spring-600'],
+  'secondary-darker':           color['spring-700'],
+  'secondary-darkest':          color['spring-800'],
+  'secondary-light':            color['spring-700'],
```
Comment (line47) [inline packages/design-system-tokens/src/themes/healthcare.ts:77]:
```suggestion
  'secondary-light':            color['spring-300'],
```

File: packages/docs/content/foundation/theme-colors.mdx [hunk 3]
```diff
@@ -57,7 +57,11 @@ The guidelines indicate how colors should be applied, noting where some should b
 
 <ColorExampleList
   colorNames={[
+    'primary-lightest',
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:60]: Added primary swatches

File: packages/docs/content/foundation/theme-colors.mdx [hunk 4]
```diff
@@ -134,6 +138,21 @@ Be cautious using these colors to signify something that may contradict what mos
 
 </ThemeContent>
 
+### Informational
+
+<ColorExampleList
```
Comment (line47) [inline packages/docs/content/foundation/theme-colors.mdx:143]: Added info color set

File: packages/design-system-tokens/src/themes/healthcare.ts (hunk 2 above)
Comment (pwolfert) [reply packages/design-system-tokens/src/themes/healthcare.ts:77]: Good catch!

File: packages/docs/content/foundation/theme-colors.mdx (hunk 4 above)
Comment (pwolfert) [reply packages/docs/content/foundation/theme-colors.mdx:143]: Hey, the theme colors pages look really good now. They each make a lot of sense to me. Thank you!

**Review by line47 — APPROVED** [review]
LGTM!

**Pull request merged**